deposit_contract.sol

```diff
@@ -87,12 +83,12 @@ contract DepositContract is IDepositContract, ERC165 {
         bytes32 deposit_data_root
     ) override external payable {
         // Extended ABI length checks since dynamic types are used.
-        require(pubkey.length == PUBKEY_LENGTH, "DepositContract: invalid pubkey length");
-        require(withdrawal_credentials.length == WITHDRAWAL_CREDENTIALS_LENGTH, "DepositContract: invalid withdrawal_credentials length");
-        require(signature.length == SIGNATURE_LENGTH, "DepositContract: invalid signature length");
+        require(pubkey.length == 48, "DepositContract: invalid pubkey length");
+        require(withdrawal_credentials.length == 32, "DepositContract: invalid withdrawal_credentials length");
+        require(signature.length == 96, "DepositContract: invalid signature length");
 
         // Check deposit amount
-        require(msg.value >= MIN_DEPOSIT_AMOUNT, "DepositContract: deposit value too low");
+        require(msg.value >= 1 ether, "DepositContract: deposit value too low");
         require(msg.value % GWEI == 0, "DepositContract: deposit value not multiple of gwei");
         uint deposit_amount = msg.value / GWEI;
```

Comment:
Should we change these to `1e9`? I think `GWEI` is fine. I wish `1 gwei` would work, but that will only come with Solidity 0.7.0.

Reply:
I think GWEI is more informative too